src/workloads/execution/LookupSBEPushdownMisc.yml

```diff
@@ -0,0 +1,358 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload conducts testing for the following test matrix:
+  - LocalCollectionCardinality: 100k / 300k / 500k / 700k
+  - DocumentSize: Small document size vs Large document size
+  - JoinFieldType: string type / sub-field type / array type
+  - NumberOfLookups: 1 / 2 / 3
+
+  But this workload does not try to measure performances for full combination of the above test
+  dimensions to avoid too lengthy test execution.
+
+  While measuring performances, this workload collects numbers for both the SBE HJ and the classic
```

Comment:
I'd encourage simplifying the workload to just do one at a time, and we can cover the classic versus SBE with different variants or patch tests as needed.